src/foam/u2/wizard/wizardflow/RemoveWizardlets.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard.wizardflow',
+  name: 'EditWizardlet',
+  implements: ['foam.core.ContextAgent'],
+
+  properties: [
+    {
+      class: 'String',
+      name: 'wizardletIds'
+    }
+  ],
+
+  methods: [
+    async function execute(x) {
```

Comment:
Does this need to be async?

Reply:
the interface is async, so it feels odd not to have it return a promise. It happens to work either way because `await` doesn't care, but I think `.then` should be expected to work on any context agent.